galaxy_importer/utils/spdx_licenses.py

```diff
@@ -0,0 +1,49 @@
+import json
+import logging
+
+from pkg_resources import resource_filename
+
+
+log = logging.getLogger(__name__)
+
+# use _get_spdx() to ref the spdx_license info, so it
+# only loaded from disk once
+_SPDX_LICENSES = None
+
+
+def _load_spdx():
+    license_path = resource_filename(__name__, 'spdx_licenses.json')
```

Comment:
The point in using resource API is that python packages are not always stored as files on raw filesystem.
For this case you can use `pkg_resources.resource_stream()` function.